src/pages/Gallery/components/Category/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as S from './category.styles'
+
+import React, { useState } from 'react'
+
+type CategoryProps = {
+  setSelectedCategory: (category: string) => void
+}
+
+const Category: React.FC<CategoryProps> = ({ setSelectedCategory }) => {
```

Comment:
```suggestion
const Category = ({ setSelectedCategory }: CategoryProps) => {
```

저희 위와 같은 컨벤션 사용하기로 했어서 위처럼 해주기!